media/js/src/AssetFilter.jsx

```diff
@@ -190,35 +195,85 @@ export default class AssetFilter extends React.Component {
         }
 
         const pagination = (
-            <nav aria-label="Page navigation example">
-                <ul className="pagination mt-2">
-                    <li className={
-                        'page-item ' +
-                            (this.state.currentPage <= 0 ? 'disabled' : '')
-                    }>
+            <div id="view-pagination" className="row">
```

Comment:
The bootstrap "blue" here does not pass accessibility standards. Marc fixed that up in [his wireframes](https://mrcrymnd.github.io/mediathread/collection-add-view-paginate.html), you can pick up the correct color there.